sdk/communication/Azure.Communication.JobRouter/samples/Sample1_HelloWorld.md

```diff
@@ -0,0 +1,91 @@
+# Azure.Communication.JobRouter Samples - Hello World (sync)
+
+## Import the namespaces
+
+```C# Snippet:Azure_Communication_JobRouter_Tests_Samples_UsingStatements
+using Azure.Communication.JobRouter;
+using Azure.Communication.JobRouter.Models;
+```
+
+## Create a client
+
+Create a `RouterClient` and send a request.
```

Comment:
Do we need the "and send a request" bit?

Reply:
removed